exporter/stats/stackdriver/stackdriver.go

```diff
@@ -309,3 +312,25 @@ func namespacedViewName(v string, escaped bool) string {
 	}
 	return path.Join("custom.googleapis.com", p)
 }
+
+func newLabels(tags []tag.Tag) map[string]string {
+	labels := make(map[string]string)
+	for _, tag := range tags {
+		labels[tag.Key.Name()] = tag.Value
+	}
+	return labels
+}
+
+func newLabelDescriptors(keys []tag.Key) []*labelpb.LabelDescriptor {
+	labelDescriptors := make([]*labelpb.LabelDescriptor, len(keys))
+	for _, key := range keys {
+		labelDescriptors = append(
+			labelDescriptors,
+			&labelpb.LabelDescriptor{
+				Key:         key.Name(),
+				ValueType:   labelpb.LabelDescriptor_STRING, // We only use string tags
+				Description: "OpenCensus TagKey",
```

Comment:
Do we need a description? If yes, I'd prefer we use "OpenCensus tag key". TagKey sounds programmatic.

Reply:
This is not a required filed, I think it's safe to omit it for now.